### PR TITLE
NCL-3175 Fix autoscroll on live log

### PR DESCRIPTION
**Pull request by matedo1**

### CODE REVIEW

File: ui/app/build-records/directives/pnc-live-log/pncLogCanvas.js [hunk 1]
```diff
@@ -57,32 +53,26 @@
...
-      startPosition = parent.scrollTop;
 
       // Catch scroll events so we can enable / disable scrolling based
       // on whether the user is scrolled to the bottom.
       parent.onscroll = function () {
-        var endPosition = parent.scrollTop;
-        var height = $element.innerHeight();
-        var bottom = parent.scrollHeight;
+        // user does not have to reach the bottom exactly, if he is close enough,
+        // autoscroll will be also activated
```
Comment (alexcreasy) [inline ui/app/build-records/directives/pnc-live-log/pncLogCanvas.js]: Nice!

**Comment by alexcreasy** [comment]
@matedo1 this looks good to me and aside from some minor changes to keep the component generic +1

**Comment by matedo1** [comment]
retest this please

**Comment by matedo1** [comment]
retest this please

**Comment by matedo1** [comment]
retest this please

**Pull request merged**